preseed/provision.erb

```diff
@@ -141,7 +141,8 @@ d-i grub-installer/only_debian boolean true
 d-i grub-installer/with_other_os boolean true
 <% if @host.params['install-disk'] -%>
 d-i grub-installer/bootdev string <%= @host.params['install-disk'] %>
-<% else -%>
+<% end -%>
+<% if @host.operatingsystem.name == 'Debian' and @host.operatingsystem.major.to_i == 8 -%>
```

Comment:
It seems that Vivid has grub-installer (1.78ubuntu25) which doesn't support "default".

Refs:
https://code.launchpad.net/~ubuntu-branches/ubuntu/vivid/grub-installer/vivid
https://tracker.debian.org/media/packages/g/grub-installer/changelog-1.103